### DISCUSSION TITLE
How does tdcs optimization apply to tacs？

### DISCUSSION THREAD

**Discussion by Jaycob-jh** [simnibs/simnibs#351] [8. Simulation workflows] [answered]
Hi, Simnibs team. I have a question about how to modify code for tacs in tdcs optimize for multi-electrode applications. In fact, I want to use tacs, where one electrode is positive current and the remaining three are negative current and used for the target area. However, when I used the tdcs optimize tutorial, the result was two positive currents and two negative currents.

**Comment by oulap** [comment]
Hi,

Yes, the tdcs optimization will find the positive and negative currents to focally stimulate the target region. At this point you can't decide how many electrodes have a positive current and how many have a negative current.

**Comment by enricovara** [comment]
If it helps highlight the relevance of this potential future feature, I also need to optimise a 3-by-1 setup where 3 electrodes have a fixed and equal current and the return electrode must equal the sum of the others.
Thanks! :)

**Comment by Dr-QingHe** [comment]
Yes, it is very necessary for the configuration to model the functions of (HD) tACS electric fields, like N-by-1 tACS, and we especially hope that future versions will include these features

**Reply by TorgeW** [reply, marked as answer]
The new version, including these features, is released now :) 

https://simnibs.github.io/simnibs/build/html/tutorial/tes_flex_opt.html#tes-flex-opt
